packages/terra-dialog/src/Dialog.scss

```diff
@@ -0,0 +1,12 @@
+:local {
+  .dialog {
+    height: 100%;
+    width: 100%;
+  }
+
+  .header,
```

Comment:
I'm thinking we should drop the padding the from the header, and possible the body and footer. With padding on the header, if we pass an action header in this prop, it will not be flush with the edges which is likely want UX intends.

Reply:
Or maybe we need a prop to toggle if padding is set on each container, I can see cases where it would be nice to have this default padding and cases where it will make it challenging to get what UX intends.